meta-xfstk/recipes-xfstk/xfstk/xfstk_1.8.1.bb

```diff
@@ -0,0 +1,26 @@
+DESCRIPTION = "Build XFSTK Downloader for recovery"
+LICENSE = "LGPL-2.1"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/LGPL-2.1;md5=1a6d268fd218675ffea8be556788b780"
+
+inherit cmake
+
+BBCLASSEXTEND = "native"
+
+DEPENDS = "g++ qtbase5 qtcreator build-essential devscripts libxml2-dev alien doxygen graphviz libusb-dev libboost-all-dev"
```

Comment:
I changed this to DEPENDS = "libxml2-native libusb-native boost-native"
But I think we have introduced a layer dependency to `meta-qt5` `qtbase`

Reply:
Is there a way to disable building qt5 version for this package?

Reply:
> I changed this to DEPENDS = "libxml2-native libusb-native boost-native"

Do we need to add the `-native` suffix? It looks that it is added automatically when building `xfstk-native` .

> But I think we have introduced a layer dependency to `meta-qt5` `qtbase`

Yup, should I add it to setup.sh so that it will be pulled, or should it be added elsewhere?

> Is there a way to disable building qt5 version for this package?

Not as far as I am aware. A lot of the individual cmake config files have qt5 as a requirements, and so it will not configure without it present.

Reply:
> > I changed this to DEPENDS = "libxml2-native libusb-native boost-native"
> 
> Do we need to add the `-native` suffix? It looks that it is added automatically when building `xfstk-native` .

I thought not. But then I searched for libxml2 under out/linux64/build/tmp/work/x86_64-linux/ and couldn't find it. And found it under out/linux64/build/tmp/work/corei7-64-poky-linux/. So that doesn't seem right. But I'm not sure.

Reply:
I think this could be just when it's being built that it's placed there. For me, at least so far, it has been building with just the recipe name.

Reply:
Like I said, I'm not really an expert... but

corei7-64-poky-linux is the target. Imagine when the target is arm. We want xfstk running on host. 
In fact I am wondering if -native is correct, it might be needed to build as sdk, like other standalone build tools ([example](https://github.com/edison-fw/meta-intel-edison/blob/master/meta-arduino/recipes-core/meta/arduino-toolchain.bb)?).

Reply:
unfortunately I am not sure if `-native` is the correct way forward, as I have not done much with bitbake.
At least for now, as long as we can get it to build, it should be okay to change it later.